decentraland/unity-explorer
# fix: various fixes regarding logout flow

## What does this PR change?

This PR fixes a series of issues related to the logout flow followed by the connection with a new account:
* our own nametag now updates correctly name and color
* backpack shows the correct equipped wearables
* in world avatar now shows the correct equipped wearables

...

## How to test the changes?



1. Launch the explorer
2. Login with one session
3. Logout
4. Login with a new session with a different account
5. Verify that the above points are working correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
